port/zos390/omrvmem.c

```diff
@@ -1296,29 +1299,40 @@ omrvmem_default_large_page_size_ex(struct OMRPortLibrary *portLibrary, uintptr_t
 	}
 #endif /* OMR_ENV_DATA64 */
 
-	while (0 != PPG_vmem_pageSize[index]) {
-		if (0 != (OMRPORT_VMEM_MEMORY_MODE_EXECUTE & mode)) {
-			if ((ONE_M == PPG_vmem_pageSize[index]) &&
-				(0 != (OMRPORT_VMEM_PAGE_FLAG_PAGEABLE & PPG_vmem_pageFlags[index]))
-			) {
-				break;
+
+	/* Determine maximum pageable, and fixed large pages available. */
```

Comment:
this comment does not help, honestly

Reply:
Thanks Dmitri, this was remnants of the previous implementation.

Reply:
Done. See 923728f8c7f40a80bc81474ea4fca1b85c26ea21